src/tracks/InstrumentTrack.cpp

```diff
@@ -771,7 +771,8 @@ void InstrumentTrack::saveTrackSpecificSettings( QDomDocument& doc, QDomElement
 
 void InstrumentTrack::loadTrackSpecificSettings( const QDomElement & thisElement )
 {
-	silenceAllNotes( true );
+	silenceAllNotes(!(m_previewMode && m_instrument && m_instrument->nodeName() == "vestige"
+					&& getInstrumentName(thisElement) == "vestige"));
```

Comment:
Any reason for this change? I don't understand why did you add this exception.

Reply:
That's because for `m_instrument` deleting there was `silenceAllNotes(true)`. But when you want not delete `m_instrument` you need `silenceAllNotes(false)` otherwise instrument is not reusable. I don't quite understand how `silenceAllNotes()` works, maybe it should be done in a different way.